lib/ansible/module_utils/facts.py

```diff
@@ -2498,6 +2498,27 @@ def get_virtual_facts(self):
                 if 'VirtualBox' in line:
                     self.facts['virtualization_type'] = 'virtualbox'
                     self.facts['virtualization_role'] = 'guest'
+        # Detect domaining on Sparc hardware
+        if os.path.exists("/usr/sbin/virtinfo"):
```

Comment:
it might be better to use the module method to find binaries, in case someone moves this to another path